registry/storage/driver/base/base.go

```diff
@@ -89,8 +97,16 @@ func (base *Base) setDriverName(e error) error {
 
 // GetContent wraps GetContent of underlying storage driver.
 func (base *Base) GetContent(ctx context.Context, path string) ([]byte, error) {
-	ctx, done := dcontext.WithTrace(ctx)
-	defer done("%s.GetContent(%q)", base.Name(), path)
+	attrs := []attribute.KeyValue{
+		attribute.String(tracing.AttributePrefix+"storage.driver.name", base.Name()),
```

Comment:
[Good call with making `storage.driver` a namespace](https://opentelemetry.io/docs/specs/semconv/general/attribute-naming/#:~:text=Names%20SHOULD%20NOT,in%20the%20future.) ✅